tools/README.md

```diff
@@ -241,29 +241,29 @@ We can input multiple Spark event logs and this tool can compare environments, e
 ### A. Compare Information Collected ###
 Compare Application Information:
 
-+--------+-----------------------+-------------+-------------+--------+-----------+------------+-------+
-|appIndex|appId                  |startTime    |endTime      |duration|durationStr|sparkVersion|gpuMode|
-+--------+-----------------------+-------------+-------------+--------+-----------+------------+-------+
-|1       |app-20210329165943-0103|1617037182848|1617037490515|307667  |5.1 min    |3.0.1       |false  |
-|2       |app-20210329170243-0018|1617037362324|1617038578035|1215711 |20 min     |3.0.1       |true   |
-+--------+-----------------------+-------------+-------------+--------+-----------+------------+-------+
++--------+-----------+-----------------------+-------------+-------------+--------+-----------+------------+-------+
+|appIndex|appName    |appId                  |startTime    |endTime      |duration|durationStr|sparkVersion|gpuMode|
++--------+-----------+-----------------------+-------------+-------------+--------+-----------+------------+-------+
+|1       |Spark shell|app-20210329165943-0103|1617037182848|1617037490515|307667  |5.1 min    |3.0.1       |false  |
+|2       |Spark shell|app-20210329170243-0018|1617037362324|1617038578035|1215711 |20 min     |3.0.1       |true   |
++--------+-----------+-----------------------+-------------+-------------+--------+-----------+------------+-------+
 ```
 
 - Compare Executor information:
 ```
 Compare Executor Information:
-+--------+----------+----------+-----------+------------+-------------+--------+--------+--------+------------+--------+--------+
-|appIndex|executorID|totalCores|maxMem     |maxOnHeapMem|maxOffHeapMem|exec_cpu|exec_mem|exec_gpu|exec_offheap|task_cpu|task_gpu|
-+--------+----------+----------+-----------+------------+-------------+--------+--------+--------+------------+--------+--------+
-|1       |0         |4         |13984648396|13984648396 |0            |null    |null    |null    |null        |null    |null    |
-|1       |1         |4         |13984648396|13984648396 |0            |null    |null    |null    |null        |null    |null    |
++--------+------------+----------------+-----------+------------+-------------+--------+--------+--------+------------+--------+--------+
+|appIndex|numExecutors|coresPerExecutor|maxMem     |maxOnHeapMem|maxOffHeapMem|exec_cpu|exec_mem|exec_gpu|exec_offheap|task_cpu|task_gpu|
```

Comment:
we need to print resource profile id, I think we should also clarify the other columns.. exec_gpu for instance perhaps that is just numGpusPerExecutor or something more readable.  how is exec_cpu different from totalCores?  Perhaps we need to make it more clear about resource profiles only being parsable when processing with spark 3.1+ as well.

Reply:
Updated.  We are printing resource profile id only if we are processing Spark3.1+ event logs.